lightspeed-stack.yaml

```
@@ -12,10 +12,10 @@ llama_stack:
   # The instance would have already been started with a llama-stack-run.yaml file
   use_as_library_client: false
   # Alternative for "as library use"
-  # use_as_library_client: true
-  # library_client_config_path: <path-to-llama-stack-run.yaml-file>
-  url: http://llama-stack:8321
-  api_key: xyzzy
+  use_as_library_client: true
```

YAMLlint (1.38.0)

[error] duplication of key "use_as_library_client" in mapping

(key-duplicates)